packages/fxa-settings/src/components/Profile/index.test.tsx

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { Profile } from '.';
+import {
+  renderWithRouter,
+  MockedCache,
+  MOCK_ACCOUNT,
+} from '../../models/_mocks';
+
+// set this here to bypase ts null warning
+let displayName = 'John Dope';
+if (MOCK_ACCOUNT.displayName) {
+  displayName = MOCK_ACCOUNT.displayName;
+}
+
+// todo:
+// add test cases for different states, including secondary email
+describe('Profile', () => {
+  it('renders "fresh load" <Profile/> with correct content', async () => {
```

Comment:
This test fails for me. You too?
![Screen Shot 2020-08-19 at 5 01 37 PM](https://user-images.githubusercontent.com/6392049/90689250-c25d5380-e23d-11ea-9d9b-d5e805791a28.png)

Reply:
I think this is related to the change with `renderWithRouter` maybe in the stories that lauren mentioned? Looking at it